sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -1430,9 +1430,13 @@ object Limit {
 /**
  * A global (coordinated) limit. This operator can emit at most `limitExpr` number in total.
  *
+ * We can not make it inherit [[OrderPreservingUnaryNode]] due to the different strategy of
+ * physical plan. The output ordering of child will be broken if a shuffle exchange comes in
+ * between the child and global limit.
+ *
  * See [[Limit]] for more information.
```

Comment:
I think "We can ..." should be after "See [[Limit]] for more information."? It reads like there are more info about `OrderPreservingUnaryNode` in `Limit`.